pkg/ddevapp/config.go

```diff
@@ -990,11 +990,11 @@ ENV N_INSTALL_VERSION="%s"
 			extraWebContent = extraWebContent + "\n" + fmt.Sprintf(`
 ### Drupal 11+ requires a minimum sqlite3 version (3.45 currently)
 ARG SQLITE_VERSION=%s
-RUN ( mkdir -p /tmp/sqlite3 && \
+RUN log-stderr.sh bash -c "mkdir -p /tmp/sqlite3 && \
```

Comment:
For next time, this would be a great place to try out the docker HEREDOC technique. Wow, is this hard to read :)